.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "semi": false,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "arrowParens": "avoid"
```

80 characters isn't enough for our big screen I'd set it to 120 at least.

```suggestion
  "arrowParens": "avoid",
  "printWidth": 120
```